.github/workflows/lint.yml

```diff
@@ -8,9 +8,23 @@ jobs:
     container:
       image: registry.fedoraproject.org/fedora:latest
     steps:
+    - uses: actions/checkout@v2
     - name: Install tools
       run: sudo dnf -y install git make python3-flake8 ShellCheck clang-tools-extra which findutils
```

Comment:
perhaps we could remove `ShellCheck`?

Reply:
I also thought about it. I see that Fedora rawhide has 0.8.0 already (https://src.fedoraproject.org/rpms/ShellCheck) but who knows if it will make its way to `fedora:latest`. I was using the Fedora container for `lint` to have up to date versions of the different tools so that I do not have to download it from "random" (not really random) places off the internet. But that is probably me just being old. I would prefer to use packages from the distribution, but if 0.8.0 does not appear really soon in `fedora:latest` than downloading it makes sense and could be removed here.

Reply:
We'll have to wait until F37. Added the TODO to switch back to distro packaged shellcheck to the yaml.